src/magic-link/magic-link.service.ts

```diff
@@ -20,7 +20,16 @@ export class MagicLinkService {
   }
 
   async getEmailFromToken(didToken: string): Promise<string> {
-    this.magic.token.validate(didToken);
+    try {
+      this.magic.token.validate(didToken);
```

Comment:
Do we need to wrap this? Isn't there an additional error catch from the caller? If it's an error from Magic, we can probably just check codes from [here](https://github.com/magiclabs/magic-admin-js/blob/master/src/core/sdk-exceptions.ts)